sormas-backend/src/main/java/de/symeda/sormas/backend/infrastructure/country/CountryFacadeEjb.java

```diff
@@ -198,37 +193,36 @@ public long count(CountryCriteria criteria) {
 	}
 
 	@Override
-	public CountryDto save(@Valid CountryDto dto) throws ValidationRuntimeException {
-		return save(dto, false);
-	}
-
-	@Override
-	public CountryDto save(@Valid CountryDto dto, boolean allowMerge) throws ValidationRuntimeException {
+	public CountryDto save(@Valid CountryDto dtoToSave, boolean allowMerge) throws ValidationRuntimeException {
 		checkInfraDataLocked();
-		
-		if (StringUtils.isBlank(dto.getIsoCode())) {
+
+		if (StringUtils.isBlank(dtoToSave.getIsoCode())) {
 			throw new EmptyValueException(I18nProperties.getValidationError(Validations.importCountryEmptyIso));
 		}
 
-		Country country = service.getByUuid(dto.getUuid());
+		Country country = service.getByUuid(dtoToSave.getUuid());
 
 		if (country == null) {
-			Optional<Country> byIsoCode = service.getByIsoCode(dto.getIsoCode(), true);
-			Optional<Country> byUnoCode = service.getByUnoCode(dto.getUnoCode(), true);
+			Optional<Country> byIsoCode = service.getByIsoCode(dtoToSave.getIsoCode(), true);
+			Optional<Country> byUnoCode = service.getByUnoCode(dtoToSave.getUnoCode(), true);
 			if (byIsoCode.isPresent() || byUnoCode.isPresent()) {
 				if (allowMerge) {
-					country = byIsoCode.isPresent() ? byIsoCode.get() : byUnoCode.get();
-					CountryDto dtoToMerge = getCountryByUuid(country.getUuid());
-					dto = DtoHelper.copyDtoValues(dtoToMerge, dto, true);
+					// todo no merging?
```

Comment:
Can this be clarified and fixed? Or at least documented in a ticket so it doesn't go unnoticed

Reply:
Follow up issue:  #6880